src/components/readArticle.jsx

```diff
@@ -2,6 +2,8 @@ import React, { Component, Fragment } from 'react';
 import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 import ReactHtmlParser from 'react-html-parser';
+import '../../node_modules/@fortawesome/fontawesome-free/js/all';
```

Comment:
Why import this from node_modules?

Reply:
Fixed now, thank you for the feedback @ezrqnkemboi